<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterWidget</class>
 <widget class="QWidget" name="ParameterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>204</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>0</height>
   </size>
  </property>
  <widget class="QLabel" name="default_value_lbl">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>53</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Default Value:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="default_value_edt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mode_lbl">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>129</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Mode:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>129</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="in_out_hlt">
    <property name="spacing">
     <number>4</number>
    </property>
    <item>
     <widget class="QCheckBox" name="param_in_chk">
      <property name="text">
       <string>IN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="param_out_chk">
      <property name="text">
       <string>OUT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="param_variadic_chk">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>VARIADIC</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
